packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -13,6 +13,7 @@ type AppInitSDKProps = {
   configurations: {
     apiDiscovery?: InitAPIDiscovery;
     appFetch: UtilsConfig['appFetch'];
+    wsConfigs: UtilsConfig['wsConfigs'];
```

Comment:
FWIW - I didn't immediately see `wsConfigs` and think `websockets`. Not sure if we could name this a bit better? Just a nit.

Reply:
🤔 
`ws:` is a web socket protocol... `wss:` is the secure version... just like `http:` and `https:`... not sure I can agree with you on this one Bryan... but I'll think on it. If you have a better name I can go with -- I don't mind changing.

I'd like to avoid `webSocketConfig` though...

Reply:
Docs might be our answer here. Maybe what's bothering me more is we have `Configs` nested in a `configurations` object.

Reply:
An interesting point. Configurations for the AppInitSDK can have configurations for web sockets though -- maybe I'm bias, but I'm not seeing a real issue here. 

I'll call it settings if you want. It's a configuration for the WSFactory... but Settings is more-or-less swappable.